src/index.js

```diff
@@ -1,21 +1,25 @@
 import createThemeProvider from './create-theme-provider';
 import createWithTheme from './create-with-theme';
+import createThemeListener from './create-theme-listener';
 import defaultChannel from './channel';
 
 export const channel = defaultChannel;
 export const withTheme = createWithTheme();
 export const ThemeProvider = createThemeProvider();
+export const themeListener = createThemeListener();
```

Comment:
Question: How can these exports be used since they all require a channel but they are invoked without one?